src/components/NcActionInput/NcActionInput.vue

```diff
@@ -84,23 +85,33 @@ For the multiselect component, all events will be passed through. Please see the
 					v-on="$listeners" />
 
 				<template v-else>
-					<input :id="id" type="submit" class="action-input__submit">
-
-					<input :type="type"
-						:value="value"
-						:placeholder="text"
-						:disabled="disabled"
-						:aria-label="ariaLabel"
-						v-bind="$attrs"
-						:class="{ focusable: isFocusable }"
-						class="action-input__input"
-						@input="onInput"
-						@change="onChange">
-					<!-- allow the custom font to inject a ::before
-						not possible on input[type=submit] -->
-					<label v-show="!disabled" :for="id" class="action-input__label">
-						<ArrowRight :size="20" />
-					</label>
+					<div class="action-input__container">
+						<label v-if="label !== undefined"
```

Comment:
```suggestion
						<label v-if="label"
```
Otherwise would be displayed with `null`, or empty strings